media-video/motion/motion-4.3.1.ebuild

```diff
@@ -0,0 +1,104 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+inherit autotools readme.gentoo-r1 systemd
+
+DESCRIPTION="A software motion detector"
+HOMEPAGE="https://motion-project.github.io"
+SRC_URI="https://github.com/Motion-Project/${PN}/archive/release-${PV}.tar.gz -> ${PN}-release-${PV}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~hppa ~ia64 ~ppc ~ppc64 ~sparc ~x86"
+IUSE="ffmpeg libav mariadb mmal mysql postgres sqlite supervise-daemon v4l webp"
+
+RDEPEND="
+	acct-group/motion
+	acct-user/motion
+	virtual/jpeg:=
+	ffmpeg? (
+		libav? ( media-video/libav:= )
+		!libav? ( media-video/ffmpeg:0= )
+	)
```

Comment:
libav has been removed from ::gentoo tree.

Reply:
Ok, I'll change it to ffmpeg? ( media-video/ffmpeg:0= )